[2023-09-11 12:00:29,878] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: listing_4_15.extract_gz scheduled__2023-09-11T05:00:00+00:00 [queued]>
[2023-09-11 12:00:29,895] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: listing_4_15.extract_gz scheduled__2023-09-11T05:00:00+00:00 [queued]>
[2023-09-11 12:00:29,896] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-09-11 12:00:29,896] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2023-09-11 12:00:29,896] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-09-11 12:00:29,918] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): extract_gz> on 2023-09-11 05:00:00+00:00
[2023-09-11 12:00:29,924] {standard_task_runner.py:52} INFO - Started process 1274 to run task
[2023-09-11 12:00:29,928] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'listing_4_15', 'extract_gz', 'scheduled__2023-09-11T05:00:00+00:00', '--job-id', '498', '--raw', '--subdir', 'DAGS_FOLDER/listing_4_15.py', '--cfg-path', '/tmp/tmp1_8tk4tc', '--error-file', '/tmp/tmp6j6wip99']
[2023-09-11 12:00:29,928] {standard_task_runner.py:80} INFO - Job 498: Subtask extract_gz
[2023-09-11 12:00:30,005] {task_command.py:371} INFO - Running <TaskInstance: listing_4_15.extract_gz scheduled__2023-09-11T05:00:00+00:00 [running]> on host b18943745c0d
[2023-09-11 12:00:30,100] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=listing_4_15
AIRFLOW_CTX_TASK_ID=extract_gz
AIRFLOW_CTX_EXECUTION_DATE=2023-09-11T05:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-09-11T05:00:00+00:00
[2023-09-11 12:00:30,101] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-09-11 12:00:30,102] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'gunzip --force /tmp/wikipageviews.gz']
[2023-09-11 12:00:30,113] {subprocess.py:85} INFO - Output:
[2023-09-11 12:00:30,697] {subprocess.py:92} INFO - 
[2023-09-11 12:00:30,698] {subprocess.py:92} INFO - gzip: /tmp/wikipageviews.gz: invalid compressed data--format violated
[2023-09-11 12:00:30,709] {subprocess.py:96} INFO - Command exited with return code 1
[2023-09-11 12:00:30,725] {taskinstance.py:1909} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 195, in execute
    f'Bash command failed. The command returned a non-zero exit code {result.exit_code}.'
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2023-09-11 12:00:30,728] {taskinstance.py:1420} INFO - Marking task as FAILED. dag_id=listing_4_15, task_id=extract_gz, execution_date=20230911T050000, start_date=20230911T120029, end_date=20230911T120030
[2023-09-11 12:00:30,743] {standard_task_runner.py:97} ERROR - Failed to execute job 498 for task extract_gz (Bash command failed. The command returned a non-zero exit code 1.; 1274)
[2023-09-11 12:00:30,784] {local_task_job.py:156} INFO - Task exited with return code 1
[2023-09-11 12:00:30,826] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
